test/ConfirmationButton_test.jsx

```diff
@@ -0,0 +1,52 @@
+/* eslint func-names: "off" */
+
+import assert from "assert";
+import React from "react";
+import sinon from "sinon";
+import {mount, shallow} from "enzyme";
+import {Button, ModalButton, ConfirmationButton} from "../src";
+
+describe("ConfirmationButton", () => {
+  it("renders a modal with confirm and cancel buttons", () => {
+    const confirmationButton = shallow(<ConfirmationButton value="Confirm pl0x" modalTitle="A title">
+      <p>Hello</p>
+    </ConfirmationButton>);
```

Comment:
nit: we've been doing indentation in other tests like this https://github.com/Clever/components/blob/master/test/Modal_test.jsx#L9-L13
```
shallow(
  <ConfirmationButton> 
   <p>Hello</p>
  </ConfirmationButton>
); 
```